action.yml

```diff
@@ -0,0 +1,19 @@
+# Copyright Kani Contributors
+# SPDX-License-Identifier: Apache-2.0 OR MIT
+
+name: 'Kani Model Checker'
+description: 'Run the Kani Model Checker on a Cargo Crate'
```

Comment:
Can you replace "Kani Model Checker" with "Kani Rust Verifier"? We don't use "Kani Model Checker" anywhere.

Reply:
06755ef4508250152023d99bed7632600f29bfb8